osquery/core/init.cpp

```diff
@@ -273,18 +287,25 @@ Initializer::Initializer(int& argc, char**& argv, ToolType tool)
         boost::filesystem::create_directory(homedir, ec)) {
       // Only apply user/shell-specific paths if not overridden by CLI flag.
       if (Flag::isDefault("database_path")) {
+        // TODO(#2001): the path should be built in a more angostic way 
         osquery::FLAGS_database_path = homedir + "/shell.db";
```

Comment:
Can this not just be changed to the following?
`osquery::FLAGS_database_path = homedir / "shell.db";`